frontend/src/actionCreators/usersActionCreator.tsx

```diff
@@ -110,4 +110,34 @@ export const getUsersAction = (params: API.IPaginateParams) => (dispatch: Functi
     .finally(() => dispatch(hideLoading()));
 };
 
+export const fetchUserDetail = (id: API.IUserDetailParams) => (dispatch: Function) => {
+  dispatch(request(reducerTypes.GET_USER_DETAIL));
```

Comment:
Please check with Devin as he moved some/all of the action types from `reducerTypes` to `actionTypes`. 
Didn't fully understood that refactor (I guess it puts the types in one file vs several) but please check with him so we use consistent patters in the app. 

@devinleighsmith could you please follow up?

Reply:
Yeah, so there are a lot of constants that were in reducerTypes that weren't reducers. IE, there are some action creators that don't have a dedicated reducer - mostly put and post requests. 

In this case, I'd say that once again GET_USER_DETAIL is actually the action here, the reducer comes into play while storing the user.